manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md

```diff
@@ -208,7 +493,7 @@ To switch an index’s lifecycle policy, follow these steps:
 
     For example, the [`forcemerge`](elasticsearch://reference/elasticsearch/index-lifecycle-actions/ilm-forcemerge.md) action temporarily closes an index before reopening it. Removing an index’s {{ilm-init}} policy during a `forcemerge` can leave the index closed indefinitely.
```

Comment:
We might want to weaken this statement to:
```suggestion
    For example, in certain cases the [`forcemerge`](elasticsearch://reference/elasticsearch/index-lifecycle-actions/ilm-forcemerge.md) action temporarily closes an index before reopening it. Removing an index’s {{ilm-init}} policy during a `forcemerge` can leave the index closed indefinitely.
```

Reply:
Fixed in [a9ebfce](https://github.com/elastic/docs-content/pull/1906/commits/a9ebfce8ed91232b8c5948f49f2bd22aaa4178d9)